src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  selectedTodo: Todo;
+  setSelectedTodo: React.Dispatch<React.SetStateAction<'' | Todo>>;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [loader, setLoader] = useState(true);
+  const [user, setUser] = useState<User | ''>('');
```

Comment:
Consider initializing `user` with `null` or an appropriate default value instead of an empty string to avoid type issues.